src/index.js

```diff
@@ -10,24 +10,36 @@ export * from './queryParser'
 import express from 'express'
 import bodyParser from 'body-parser'
 
-function makeHandler (handler) {
+function makeHandler (handler, extendedHandlers) {
   return function (req, res, next) {
-    return utils.resolve()
-      .then(function () {
-        return handler(req)
-      })
-      .then(function (result) {
-        res.status(200)
-        if (!utils.isUndefined(result)) {
-          res.send(result)
-        }
-        res.end()
-      })
-      .catch(next)
+    return utils.resolve().then(function () {
+      if (extendedHandlers && extendedHandlers.req) {
+        return new Promise((resolve) => {
+          extendedHandlers.req(req, res, resolve)
+        })
+        .then(() => { return handler(req) })
+        .catch(next)
+      } else if (extendedHandlers === false) {
+        return Promise.reject()
```

Comment:
`extendedHandler === false` blocks an endpoint - ie: If you don't want the delete endpoint available via `js-data-express` you would just set `false` and it returns `404`.